jormungandr/src/settings/start/mod.rs

```diff
@@ -92,6 +83,45 @@ impl RawSettings {
             });
         }
 
+        // If the command line specifies log arguments, check that the arg
+        // is Some(output), else, skip.
+        let cmd_output = self.command_line.log_output.clone();
+
+        if let Some(output) = cmd_output {
+            let cmd_level = self.command_line.log_level;
+            let cmd_format = self.command_line.log_format;
+
+            // Read the log settings, and look if we already
+            // have this output configured
+            for entry in entries.iter_mut() {
+                // If the output is aleady configured:
+                if &entry.output == &output {
```

Comment:
Eh, I should have specified more clearly: if the log output is set in the command line, it replaces all entries from config, rather than just modifying settings for any matching output entry.